src/ui/FileViewer/index.scss

```diff
@@ -106,3 +115,26 @@
     }
   }
 }
+
+.sendbird-file-viewer-arrow--left,
+.sendbird-file-viewer-arrow--right {
+  cursor: pointer;
+  position: absolute;
+  height: 32px;
+  width: 32px;
+  top: calc(50% - 16px);
+  &:hover {
+    border-radius: 50%;
+    background-color: rgb(98, 144, 144);
+    box-shadow: 0 0 0 2px #fff, 0 0 0 4px rgb(98, 144, 144);
```

Comment:
next pr -> should be inside `include themed`

 @include themed() {
    background-color: t(bg-0);
  }